variables.tf

```diff
@@ -1,12 +1,51 @@
-# variable "variable_name" {
-#	  description = "variable description"
-#   type        = number
-#   default     = 1
-# }
-
-# variable "variable_password" {
-#	  description = "variable description"
-#   type        = string
-#   sensitive   = true
-#   default     = "abc"
-# }
+variable "databasename" {
+  description = "The name of the database cluster."
+  type        = string
+  nullable    = false
+}
+variable "postgresql_main_node_count" {
+  description = "Number of mysql nodes that will be created."
+  type        = number
+  default     = 1
+}
+
+variable "engine" {
```

Comment:
It is postgres module, what if I use "redis" here? Saver and better to hardcode this variable in resource call.